packages/mesos-modules/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o nounset -o pipefail
+
+pushd /pkg/src/mesos-modules/
+./bootstrap
+popd
+
+export LD_LIBRARY_PATH=/opt/mesosphere/lib
```

Comment:
Rather than do this, might be worthwhile to just `source /opt/mesosphere/environment.export`, which will cause you to pick up LD_LIBRARY_PATH among other things (/opt/mesosphere/bin added to $PATH, etc). Be good to start with that.